### PR TITLE
Add Gradle for Kotlin streaming gRPC example.

### PR DESCRIPTION
**Pull request by fashing**
Also add a unit test for the gRPC Kotlin code.

### CODE REVIEW

File: examples/src/test/kotlin/org/wfanet/examples/streaming/HelloStreamingTest.kt [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        .start()
+    )
+    val channel = InProcessChannelBuilder.forName(
+      serverName
+    ).directExecutor().build()
+    client = HelloStreamingClient(grpcCleanup.register(channel))
+  }
+
+  @Test
+  fun testStuff() = runBlocking {
```
Comment (SanjayVas) [inline examples/src/test/kotlin/org/wfanet/examples/streaming/HelloStreamingTest.kt:39]: Better name? Use Kotlin standard of multiple words in backticks.

File: examples/src/test/kotlin/org/wfanet/examples/streaming/HelloStreamingTest.kt [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    grpcCleanup.register(
+      InProcessServerBuilder.forName(serverName)
+        .directExecutor()
+        .addService(HelloStreamingServer.HelloStreamingService())
+        .build()
+        .start()
+    )
+    val channel = InProcessChannelBuilder.forName(
+      serverName
+    ).directExecutor().build()
```
Comment (SanjayVas) [inline examples/src/test/kotlin/org/wfanet/examples/streaming/HelloStreamingTest.kt:34]: Odd formatting?

File: examples/src/test/kotlin/org/wfanet/examples/streaming/HelloStreamingTest.kt [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+@RunWith(JUnit4::class)
+class HelloStreamingTest {
+  @Rule
+  @JvmField
+  val grpcCleanup = GrpcCleanupRule()
+
+  lateinit var client: HelloStreamingClient
+
+  @Before
+  fun setup() {
```
Comment (SanjayVas) [inline examples/src/test/kotlin/org/wfanet/examples/streaming/HelloStreamingTest.kt:23]: nit: better name. Perhaps "initClient"?

**Review by SanjayVas — CHANGES_REQUESTED** [review]
Not familiar enough with Gradle to give any comments there.

File: examples/src/test/kotlin/org/wfanet/examples/streaming/HelloStreamingTest.kt (hunk 1 above)
Comment (fashing) [reply examples/src/test/kotlin/org/wfanet/examples/streaming/HelloStreamingTest.kt]: Done

File: examples/src/test/kotlin/org/wfanet/examples/streaming/HelloStreamingTest.kt (hunk 2 above)
Comment (fashing) [reply examples/src/test/kotlin/org/wfanet/examples/streaming/HelloStreamingTest.kt]: Done

File: examples/src/test/kotlin/org/wfanet/examples/streaming/HelloStreamingTest.kt (hunk 3 above)
Comment (fashing) [reply examples/src/test/kotlin/org/wfanet/examples/streaming/HelloStreamingTest.kt]: Done

**Review by efoxepstein — COMMENTED** [review]
LGTM -- @SanjayVas to approve

**Review by SanjayVas — APPROVED** [review]

**Comment by SanjayVas** [comment]
This is for issue #26

**Pull request merged**